src/api/config/application.rb

```diff
@@ -85,8 +85,8 @@ class Application < Rails::Application
 
     config.action_controller.perform_caching = true
 
-    # Don't use cache versioning for now
-    config.active_record.cache_versioning = false
+    # Use cache versioning
```

Comment:
[nitpick]

I would remove this comment since it's pretty obvious.

Reply:
Done.